File: ruby/spec/lib/katas/descending_order/solution_spec.rb
# frozen_string_literal: true

describe 'Solution' do
  include DescendingOrder

  it 'Fixed tests' do
    Test.assert_equals(descending_order(0), 0)
    Test.assert_equals(descending_order(1), 1)
    Test.assert_equals(descending_order(123_456_789), 987_654_321)
  end
end

File: ts/src/katas/dup_encoder/solution.spec.ts
import { duplicateEncode } from './solution.js'
import * as chai from 'chai';
import 'mocha'

const assert = chai.assert;
chai.config.truncateThreshold=0;

describe("Duplicate Encoder", () => {
  it("Testing for fixed tests", () => {
    assert.strictEqual(duplicateEncode("din"),"(((");
    assert.strictEqual(duplicateEncode("recede"),"()()()");
    assert.strictEqual(duplicateEncode("Success"),")())())","should ignore case");
    assert.strictEqual(duplicateEncode("(( @"),"))((");
  });
});

